start_dev.sh

```
@@ -1,24 +1,44 @@
 source venv/bin/activate
-python src/main.py &
+PYTHONPATH=src python src/main.py > backend.log 2>&1 &
 BACKEND_PID=$!
 echo "Backend started with PID: $BACKEND_PID"
-cd - # Go back to the previous directory
 
+# 2. Start Frontend
 echo "Starting SolSniperX Frontend..."
-cd /home/mulky/Desktop/SolSniperX/frontend
-pnpm run dev --host &
+cd "$SCRIPT_DIR/frontend"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)